lib/portal.js

```diff
@@ -9,6 +9,27 @@ const KEYCODES = {
 
 export default class Portal extends React.Component {
 
+  static defaultProps = {
+    onOpen: () => {},
+    onClose: () => {},
+    onUpdate: () => {},
+  };
+
+  static propTypes = {
+    className: React.PropTypes.string,
+    style: React.PropTypes.object,
+    children: React.PropTypes.element.isRequired,
+    target: React.PropTypes.instanceOf(window.HTMLElement),
```

Comment:
The `defaultValue` could be `target : 'body'`.
So, you try to get the element with `document.querySelector`.
If it returns `null` you create the element.
So, if the `target` prop is a selector that doesn't exist, the `portal` lib will create it and use it as target.

```javascript
<... target=".my-div" ... />

...

const { target } = this.props;
let htmlElement = document.querySelector(target);
if (!htmlElement) {
  createElement(target);  // and append to body
}
htmlElement = document.querySelector(this.props.target);
```

@luchkonikita, You'll need to implement a function that creates a element from a CSS selector. Maybe it's the hard part.

This approach seems the best way of doing it.
It fits perfectly my use case.
I'll try to do it on my fork and update here.